app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -153,20 +154,24 @@ class CasesService @Inject()(appConfig: AppConfig,
     } yield updated
   }
 
-  def cancelRuling(original: Case, operator: Operator, clock: Clock = Clock.systemDefaultZone())
+  def cancelRuling(original: Case, reason: CancelReason,
+                   operator: Operator, clock: Clock = Clock.systemDefaultZone())
                   (implicit hc: HeaderCarrier): Future[Case] = {
     val updatedEndDate = LocalDate.now(clock).atStartOfDay(appConfig.zoneId)
 
     val decisionUpdating: Decision = original.decision
       .getOrElse(throw new IllegalArgumentException("Cannot Cancel a Case without a Decision"))
-      .copy(effectiveEndDate = Some(updatedEndDate.toInstant))
+      .copy(
+        effectiveEndDate = Some(updatedEndDate.toInstant),
+        cancelReason = Some(reason)
+      )
     val caseUpdating = original.copy(status = CaseStatus.CANCELLED, decision = Some(decisionUpdating))
 
     for {
       // Update the case
       updated: Case <- connector.updateCase(caseUpdating)
       // Create the event
-      _ <- addStatusChangeEvent(original, updated, operator)
+      _ <- addStatusChangeEvent(original, updated, operator, comment = Some(CancelReason.format(reason)))
```

Comment:
It's worth checking with Joe if there is a need to ever add an extra Note when cancelling a case. We'd need to keep the comment field free for that if we do

Reply:
Or we can concat them both I guess